guides/common/modules/proc_installing-openscap-plugin.adoc

```diff
@@ -1,32 +1,27 @@
 [id="Installing_the_OpenSCAP_Plugin_{context}"]
 = Installing the OpenSCAP Plug-in
 
-You can install and enable the OpenSCAP plug-in to generate OpenSCAP compliance reports.
-The OpenSCAP plug-in consists of the main OpenSCAP plug-in itself, the OpenSCAP smart proxy plug-in, and the OpenSCAP Hammer CLI plug-in.
+You can install and enable the OpenSCAP plug-in to generate SCAP compliance reports.
+The OpenSCAP plug-in consists of the OpenSCAP {Project} plug-in, the OpenSCAP Hammer CLI plug-in, the OpenSCAP {SmartProxy} plug-in, and modules for policy deployment.
 
 .Procedure
 . Install the OpenSCAP plug-in on your {ProjectServer}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-openscap --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-puppet-module true
+# {foreman-installer} \
+--enable-foreman-plugin-openscap \
+--enable-foreman-cli-openscap \
+--enable-foreman-proxy-plugin-openscap \
+--foreman-proxy-plugin-openscap-ansible-module true \
+--foreman-proxy-plugin-openscap-puppet-module true
 ----
-ifndef::satellite[]
-. Optional: Install the OpenSCAP Hammer CLI plug-in on your {ProjectServer}:
+. Optional: Install the OpenSCAP plug-in on any {SmartProxyServers}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-cli-openscap
+# {foreman-installer} \
```

Comment:
Same os above:

````
# foreman-installer --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-ansible-module true --foreman-proxy-plugin-openscap-puppet-module true
2023-09-29 08:38:31 [NOTICE] [root] Loading installer configuration. This will take some time.
2023-09-29 08:38:34 [NOTICE] [root] Running installer with log based terminal output at level NOTICE.
2023-09-29 08:38:34 [NOTICE] [root] Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG. See --full-help for definitions.
ERROR: Unrecognised option '--foreman-proxy-plugin-openscap-ansible-module'

See: 'foreman-installer --help'
````

I am also wonding if "foreman-installer" is the proper attribute because there is also : "installer-scenario-smartproxy": https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L72

Reply:
I was told that the scenario is saved during the first installer run and therefore is not required during subsequent calls.